### PR TITLE
Clarify state of the project in readme.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Just a small hint for those who are confused by what's going on with the project, we should probably update website later as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,10 +5,14 @@
 
 ## A Specification Framework for the JVM
 
+Visit the [web site](https://jetbrains.github.io/spek) for more documentation and information
 
+## Current State of the Project
 
-Visit the [web site](https://jetbrains.github.io/spek) for more documentation and information
+- Spek 1.x is in support mode, only bug fixes and documentation updates will be merged into 1.x branch.
+- Spek 2.x is main development branch.
```
Comment (raniejade) [inline README.md]: Are we bringing back `master` at some point?

**Review by raniejade — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (artem-zinnatullin) [reply README.md]: I would say "no" for now, it's just a branch name :) Semantic versioning is easier when branches follow major version so if we would need 3.x that won't require killing master again

Comment (raniejade) [reply README.md]: :+1:

Comment (arturdryomov) [inline README.md]: Probably it would be helpful to state `2.x` goals or link to the milestone.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (artem-zinnatullin) [reply README.md]: Added link to milestone, I didn't do it because it was something more suitable for 2.x branch or website, but ok

**Comment by artem-zinnatullin** [comment]
@ming13 rePTAL :)

**Comment by ColinHebert** [comment]
Are there alpha/milestone versions of 2.x available? Should that be advertised?

**Comment by artem-zinnatullin** [comment]
@ColinHebert not yet, you can build it yourself from `2.x` branch though, we just don't want to ship something that can have breaking changes in every update and make people angry :)

**Review by arturdryomov — APPROVED** [review]

**Pull request merged**